asmcomp/selectgen.ml

```diff
@@ -288,6 +292,9 @@ method select_operation op args =
   | (Cfloatofint, _) -> (Ifloatofint, args)
   | (Cintoffloat, _) -> (Iintoffloat, args)
   | (Ccheckbound _, _) -> self#select_arith Icheckbound args
+  | (Cperform, _) -> (Iperform, args)
+  | (Cresume, _) -> (Iresume_ind, args)
```

Comment:
Yes, and it will be. I haven't implemented tail-call optimisation yet. (The _ind version is always correct, but the tail_ind version is faster / uses less space)